webapp/src/pages/Settings.tsx

```diff
@@ -164,7 +164,7 @@ const Settings: React.FC = () => {
     />
   );
 
-  const displayArgumentsList = (
+  const displayKeyArgumentsList = (
     name: string,
     argEntries: Record<string, any> | any[]
```

Comment:
I guess you want to allow for objects only, now?
```suggestion
    argEntries: Record<string, any>
```

Reply:
yes, should allow only objects for now.

Reply:
Perfect, thanks! By the way, I think you meant "from now on" and not "for now".